rmgpy/cantherm/statmech.py

```diff
@@ -692,12 +705,12 @@ def applyEnergyCorrections(E0, modelChemistry, atoms, bonds,
         # The enthalpies listed here should correspond to the allowed elements in atom_num_dict
         atomHf = {'H': 51.63,
                   'Li': 37.69, 'Be': 76.48, 'B': 136.2, 'C': 169.98, 'N': 112.53, 'O': 58.99, 'F': 18.47,
-                  'Na': 25.69, 'Mg': 34.87, 'Al': 78.23, 'Si': 106.6, 'P': 75.42, 'S': 65.66, 'Cl': 28.59}
+                  'Na': 25.69, 'Mg': 34.87, 'Al': 78.23, 'Si': 106.6, 'P': 75.42, 'S': 65.66, 'Cl': 28.59, 'I':0}
         # Thermal contribution to enthalpy Hss(298 K) - Hss(0 K) reported by Gaussian thermo whitepaper
         # This will be subtracted from the corresponding value in atomHf to produce an enthalpy used in calculating the enthalpy of formation at 298 K
         atomThermal = {'H': 1.01,
                        'Li': 1.1, 'Be': 0.46, 'B': 0.29, 'C': 0.25, 'N': 1.04, 'O': 1.04, 'F': 1.05,
-                       'Na': 1.54, 'Mg': 1.19, 'Al': 1.08, 'Si': 0.76, 'P': 1.28, 'S': 1.05, 'Cl': 1.1}
+                       'Na': 1.54, 'Mg': 1.19, 'Al': 1.08, 'Si': 0.76, 'P': 1.28, 'S': 1.05, 'Cl': 1.1, 'I':0}
```

Comment:
Can you add the true values for iodine here and adjust your atomic energies so that future model chemistries that might want to use iodine have the true values available? You can probably use the values from [here](http://www.science.uwaterloo.ca/~cchieh/cact/tools/thermodata.html) (note that the heats of formation in `atomHf` are at 0 K).